va/va_enc_av1.h

```diff
@@ -249,7 +249,10 @@ typedef struct  _VAEncSequenceParameterBufferAV1 {
      */
     uint8_t     seq_tier;
 
-    uint8_t     reserved8b;
+    /** \brief Indicates whether or not the encoding is in dyadic hierarchical GOP structure.
+     *  value range [0..1].
+     */
+    uint8_t     hierarchicalFlag;
```

Comment:
Why it's named in camel case if everything else is not? This should be:
```
uint8_t hierarchical_flag;
```

Reply:
> Why it's named in camel case if everything else is not? This should be:
> 
> ```
> uint8_t hierarchical_flag;
> ```

Thanks your carefully review, fixed.